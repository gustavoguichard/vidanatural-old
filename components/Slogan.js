import { memo } from 'react'
import classnames from 'classnames'
import ImageReplace from 'components/ImageReplace'

const Slogan = ({ className, style, responsive, ...props }) => (
  <ImageReplace
    {...props}
    style={{ ...style, ...{ width: responsive || 288, height: 45 } }}
    width="288"
    height="45"
    className={classnames('slogan', className)}
    src="slogan.png"
  >
    <h2>Eu uso</h2>
    <h3>Cosm√©tica Consciente</h3>
  </ImageReplace>
)

export default memo(Slogan)
