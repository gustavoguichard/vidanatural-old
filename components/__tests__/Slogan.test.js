import { render } from 'react-testing-library'
import Slogan from 'components/Slogan'

it('renders correctly', () => {
  const { container } = render(<Slogan foo="bar" className="test" />)
  expect(container).toMatchSnapshot()
})
