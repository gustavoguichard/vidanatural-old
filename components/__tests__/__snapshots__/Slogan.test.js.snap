// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="slogan test"
    foo="bar"
    style="display: block; background: url(/static/slogan.png) no-repeat center; background-size: contain; height: 45px; width: 288px; overflow: hidden; text-indent: -999em;"
  >
    <h2>
      Eu uso
    </h2>
    <h3>
      Cosm√©tica Consciente
    </h3>
  </div>
</div>
`;
